scripts/create_perf_json.py

```diff
@@ -2058,7 +2097,7 @@ def __init__(self, base_path: Path):
 
             # Add metric files that will be used for each model.
             files[shortname]['tma metrics'] = Path(base_path, 'TMA_Metrics-full.csv')
-            if shortname == 'ADLN':
+            if shortname in ['ADLN', 'SRF']:
```

Comment:
Add `GRR`.

Reply:
Thanks Ed, I'd missed in the LKML post:
https://lore.kernel.org/lkml/20250326000434.83325-13-irogers@google.com/
that GRR had lost TMA metrics, so I'll need to repost those with this update too. I tried to spot other occurrences of this issue in the LKML posts for different models but couldn't spot any. Let me know if I missed something. Thanks!

Reply:
The GRR metrics are now sent to LKML and there is an additional converter script patch adding the GRR support.

Reply:
Thanks Ian!